### ISSUE TITLE
Should not have to edit DSF to delete an installation

### ISSUE THREAD

**Issue by schollii** [Praqma/helmsman#129]
It makes sense to use the `enabled` flag to remove individual apps from an installation, but it very awkward to have to do that when you just want to uninstall the whole thing. In that case, there should be something like `helmsman delete RELEASE_NAME` which should do the same as though every item in the DSF had `enabled: false` AND ask user if namespace should also be deleted (this would could also be done with something like `helmsman delete --namespace`). 

Basically there should be a way to go back to the state before the install was done (within reason, of course -- if things were done to the cluster that have nothing to do with the install then that is out of scope). Like you have the DSF with all enabled true, but you have not run the install yet.

**Comment by luisdavim** [comment]
Well, the idea of helmsman is to enable you to keep track of your helm releases as code so you track changes like adding or deleting components. If you want to delete a release without keeping track of that change why not just use `helm delete --purge RELEASE_NAME`?

**Comment by sami-alajrami** [comment]
@schollii you may also use the existing `helmsman --destroy` flag 
```
--destroy
        delete all deployed releases. Purge delete is used if the purge option is set to true for the releases.
```

I will close this issue. But feel free to reopen if this still does not answer your question.

**Comment by schollii** [comment]
To answer your previous question, `helm delete --purge RELEASE_NAME` is for one chart, so in the case where I want to delete all install charts, I would have to do this manually for each one, laborious and not scalable. 

Your latest comment indicates that this is actually already possible in helmsman, just not documented: --destroy. This option solves the issue. However it should not be closed because the option is not documented AFAICT, so this should lead to a documentation fix.
